environment/src/main/java/com/sequenceiq/environment/parameters/validation/validators/parameter/AzureParameterValidator.java

```diff
@@ -124,11 +127,21 @@ private ValidationResult validateEntitlement(ValidationResultBuilder validationR
                                         + "but that feature is currently disabled").build();
                     }
                 default:
-                    return validationResultBuilder.build();
+                    break;
+            }
+        }
+
+        String keyUrl = azureResourceEncryptionParametersDto.getKeyUrl();
+        if (Objects.nonNull(keyUrl)) {
+            if (!entitlementService.isAzureDiskSSEWithCMKEnabled(accountId)) {
+                LOGGER.info("Invalid request, isAzureDiskSSEWithCMKEnabled entitlement turned off for account {}", accountId);
```

Comment:
No, the entitlement is called `CDP_CB_AZURE_DISK_SSE_WITH_CMK`. Please use that name here and also in the validation error below. Furthermore, this log shall be an `error()`.

Reply:
Shall I consider changing above log level as well for `singleResourceGroupDedicatedStorageAccountEnabled` and `singleResourceGroupDeploymentEnabled`.
Though I think, it is logged as INFO here and later passed as error to be validated by the `AzureParameterValidatorTest.java ` if error is present - no further occurrences of validation are seen.

Reply:
Ah, good point. Okay, so we can keep using `info()`, perhaps an `error()` would be an overkill. But I agree that the above two log messages should be corrected to contain the right entitlement name.